routes.js

```diff
@@ -0,0 +1,25 @@
+/* routes.js */
```

Comment:
I think the functionality of this file can be included inside of the controllers for both ldp and tesseract. Let's look at this together tomorrow